internal/app/singularity/remote_use.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package singularity
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sylabs/singularity/internal/pkg/remote"
+)
+
+// RemoteUse sets remote to use
+func RemoteUse(configFile, name string) (err error) {
+	c := &remote.Config{}
+
+	file, err := os.OpenFile(configFile, os.O_RDWR|os.O_CREATE, 0600)
+	if err != nil {
+		return fmt.Errorf("while opening remote config file: %s", err)
+	}
+	defer file.Close()
+
+	c, err = remote.ReadFrom(file)
+	if err != nil {
+		return fmt.Errorf("while parsing remote config data: %s", err)
+	}
+
+	if err := c.SetDefault(name); err != nil {
+		return err
+	}
+
+	if err := file.Truncate(0); err != nil {
```

Comment:
I would add some comments here, like:

```
// truncate the file...
```
...
```
// open file config file...
```

Just to help people (like me) understand whats going on here.

Reply:
Actually, in this case I would say that the intention of the code is clear enough from the function names.